doc/ansible/dev/developer_guide.md

```diff
@@ -312,19 +312,12 @@ foo-operator       1         1         1            1           1m
 
 #### Viewing the Ansible logs
 
-The `foo-operator` deployment creates a Pod with two containers, `operator` and `ansible`.
-The `ansible` container exists only to expose the standard Ansible stdout logs that most Ansible
-users will be familiar with. In order to see the logs from a particular container, you can run
+The `foo-operator` deployment creates a Pod a an `operator` container. In order to see the logs from a particular container, you can run:
```

Comment:
typo: "create a Pod a an `operator`" should match the other ones in this document.

Reply:
I update this are let me know if it solves your point here.